LoRaEngine/modules/LoRaBasicsStationModule/start_basestation_amd64.sh

```diff
@@ -15,6 +15,12 @@ if [[ -z "$SPI_DEV" ]] || [[ $SPI_DEV == '$LBS_SPI_DEV' ]]; then
     SPI_DEV = 0
 fi
 
+if [[ -z "$TC_TRUST_PATH" ]]; then
```

Comment:
Couldn't we default to check if a cert is certs on a path (e,.g. /etc/certs/lora) and check if we have a cert there?

Reply:
Added a DEFAULT_TC_TRUST_PATH to be "/var/lorastarterkit/certs/tc.trust" (chose /var, as I was thinking about compatibility with EFLOW)